templates/core/terraform/airlock/eventgrid_topics.tf

```diff
@@ -1,14 +1,54 @@
+
+# Event grid status_changed private endpoint
```

Comment:
Can this go in the templates/core/terraform/network/dns_zones.tf file?

Reply:
it can, but then it will make it more integrated into the tre core , which is harder to split later to a shared service...

Reply:
But if we need to use that DNS name elsewhere in another shared service we would be unable to as a DNS zone can only exist once per VNet.

Reply:
fixed